dev/Detours/disolarm64.cpp

```diff
@@ -1,2 +1,5 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
```

Comment:
You're forking Detours source and relicensing it. We shouldn't do this.

[Detours is MIT License](https://github.com/microsoft/Detours/blob/master/LICENSE.md) so it's compatible with us but we should not fork their source. We don't even want their source, just forced to as they don't provide a nuget or equivalent for VS to consume

We can drop this project if/when Detours provides a nuget. Perhaps you'd care to help make that happen instead of forking their source? https://github.com/microsoft/Detours/issues/124

Reply:
Fixed.  Removed the Detours changes and added an exception to the script.